target/linux/ramips/image/mt7621.mk

```diff
@@ -13,6 +13,12 @@ define Build/gemtek-trailer
 	printf "%s%08X" ".GEMTEK." "$$(cksum $@ | cut -d ' ' -f1)" >> $@
 endef
 
+define Build/humax-factory
+        mv $@ upg
+        $(STAGING_DIR_HOST)/bin/zip -P $(1) $@.new upg
```

Comment:
Tab for indent here.

And it seems we don't need `STAGING_DIR_HOST` here.
https://github.com/openwrt/openwrt/blob/master/include/image-commands.mk#L509
BTW perhaps it might be possible to extend `Build/zip` and use it instead of creating another one...

Reply:
Yeah, It works without `STAGING_DIR_HOST` but I added it for consistency.
Anyway I remove it!